src/ast/nodes/CallExpression.ts

```diff
@@ -33,6 +33,9 @@ export default class CallExpression
 	/** Marked with #__PURE__ annotation */
 	declare annotationPure?: boolean;
 
+	private _evalWarned = false;
+	private _namespaceWarned = false;
```

Comment:
To reduce memory overhead, we can convert this boolean flag to a bit flag, like the optional property below.